yarn.lock

```diff
@@ -1404,6 +1404,17 @@
     "@uifabric/set-version" "^7.0.23"
     tslib "^1.10.0"
 
+"@fluentui/react-compose@", "@fluentui/react-compose@^0.19.12":
```

Comment:
yarn.lock has some unexpected addition (@fluentui/* pkgs)

Reply:
strange... all I ran was `lerna add` lemme revert and try again with `yarn

Reply:
Had to nuke node_modules and remove the symlinks to try again, should be fine now